content/en/getting_started/integrations/aws.md

```diff
@@ -0,0 +1,270 @@
+
+5. From the integration tile on the Datadog side, enter the AWS account ID and AWS Role name. If the role name wasn't changed in the CloudFormation setup page, use `DatadogIntegrationRole`. Datadog automatically queries AWS with the provided values, and a successful query returns a status that tells you your credentials are valid. Scroll down to find the `Install Integration` button.
+
+6. Click `Install Integration`. For this demonstration, all options have been selected (only `Collect custom metrics` is unticked by default):
+{{< img src="getting_started/integrations/integration-tile-complete.png" alt="An image displaying the AWS integration tile from within the Datadog account. The left hand side shows that the EC2 automuting option is enabled. There is a section titled 'Limit metric collection by AWS Service' which displays the sub-integrations associated with the Datadog AWS integration. These are ApiGateway, ApplicationELB, AppRunner, AppStream, AppSync, Athena, AutoScaling, Billing, Budgeting, CertificateManager, CloudFront, CloudHSM, CloudSearch, CodeBuild, Cognito, and Connect. There is a heading which states 'Turning on sub-integrations can affect your CloudWatch API usage. See our AWS FAQ for more info.'. All boxes are displayed as checked. Below this is another section called 'Other options'. There are two checkboxes here, Collect CloudWatch alarms and Collect custom metrics. Both options are checked.' On the right hand side of the page there is a section showing the settings for the connected AWS account. The Account ID is displayed as an obfuscated value. The AWS Role name is displayed as DatadogIntegrationRole.">}}
+
+Depending on which AWS services you're using and your use case for monitoring, there are multiple options within the integration tile to specify the data to be collected. For example, you can limit data collection based on AWS service, namespace, or tags. Additionally, you can choose to mute monitor notifications, for instance terminations triggered manually or by autoscaling with [EC2 automuting][14] enabled. If needed, enable [Alarm Collection][15] to send your CloudWatch alarms to the Datadog [Event Stream][16] and choose whether to collect custom metrics.
+
+### Validation
+
+View the out-of-the-box [AWS overview dashboard][17] to see metrics sent by your AWS services and infrastructure:
+{{< img src="getting_started/integrations/aws-dashboard.png" alt="An image displaying the AWS overview dashboard in the Datadog account. On the left is the AWS logo and an AWS events graph showing 'No matching entries found'. In the center are graphs related to EBS volumes with numerical data displayed and a heat map showing consistent data. Along the right are graphs related to ELBs showing numerical data as well as a timeseries graph showing spikey data from three sources.">}}
+
+## Enable sub-integrations
+
+See the [Integrations page][18] for a full listing of the available sub-integrations.
+
+## Send logs
```

Comment:
This section needs a lot of work, or we defer to https://docs.datadoghq.com/logs/guide/send-aws-services-logs-with-the-datadog-lambda-function/?tab=awsconsole#enable-logging-for-your-aws-service

1. For the cloudtrail example, we should actually do a step by step of that with new screenshots for these steps https://docs.datadoghq.com/integrations/amazon_cloudtrail/#log-collection or nor not include since it is confusing.

3. We need an intro section about how sending logs to Datadog from AWS works overall. Everything follows these 3 general steps:
1. Setup the infrustructure for sending logs to Datadog. There are 2 options for this (Lambda forwarder OR Kinesis Data Firehose as mentioned here: https://docs.datadoghq.com/integrations/amazon_web_services/?tab=roledelegation#log-collection
2. Enable logging in AWS for each AWS service/resource you wish to generate logs. We have docs on this usually on individual integration docs pages for the different AWS integrations.
3. Set up triggers or subscription filters from the AWS log locations (CloudWatch logs or S3) to the Datadog infrastructure you set up to send logs (Lambda forwarder OR Kinesis Data Firehose

Reply:
cool, I have linked out to this list for now, but as a fast iterative update we'll update the list so we can use the same content in both places